wgpu/src/lib.rs

```diff
@@ -3106,7 +3106,8 @@ impl<'a> RenderPass<'a> {
 
     /// Sets the scissor region.
     ///
-    /// Subsequent draw calls will discard any fragments that fall outside this region.
+    /// Subsequent draw calls will discard any fragments that fall outside this region
+    /// as framebuffer space.
```

Comment:
```suggestion
    /// Subsequent draw calls will discard any fragments that fall outside this region.
```

`as framebuffer space`?

Reply:
this was a comment in the WGPU user chat about this left at https://matrix.to/#/!XFRnMvAfptAHthwBCx:matrix.org/$jYU5UBDRL6d0yqihEv6l2eKCEBTeBtvge_G4WQdTd1Y?via=matrix.org&via=mozilla.org&via=kyju.org  which was also Thumbed up by @cwfitzgerald. We are trying to find a way to show that the Clip region is based on the device coords and not world coords. As this was a confusing thing for some. 

We probably could change this in another way to make it more apparent though as i do agree that FrameBuffer space is not well defined as to what it really is as you said in your other comment.

Reply:
The WGPU Spec actually cover's this by linking to the viewport coordinates inside the wgpu specs.